### DISCUSSION TITLE
[feature] add support to bypass using % wildcard

### DISCUSSION THREAD

**Discussion by ctf0** [laravie/query-filter#9] [Ideas]
basically a way to escape 
https://github.com/laravie/query-filter/blob/b6c10c5b962314a7c446a299edb2b2b07fa700a0/src/Searchable.php#L109

so instead of always getting
```sql
where ((`rooms`.`number` like '1' or `rooms`.`number` like '1%' or `rooms`.`number` like '%1' or `rooms`.`number` like '%1%'))
```

we can only get
```sql
where ((`rooms`.`number` like '1'))
```

**Comment by crynobone** [comment]
https://github.com/laravie/query-filter#search-with-exact-wildcard 

Added in v2.1.0
